src/ocm/components/clusterConfiguration/networkConfiguration/networkConfigurationValidation.ts

```diff
@@ -59,8 +59,14 @@ export const getNetworkConfigurationValidationSchema = (
 ) =>
   Yup.lazy<NetworkConfigurationValues>((values) =>
     Yup.object<NetworkConfigurationValues>().shape({
-      apiVip: vipValidationSchema(hostSubnets, values, initialValues.apiVip),
-      ingressVip: vipValidationSchema(hostSubnets, values, initialValues.ingressVip),
+      apiVip:
+        values.apiVip !== undefined && values.apiVip !== ''
```

Comment:
With the new development for Dual Stack IPs, the API now does not allow setting only one of  `apiVip / ingressVip`, we would get an error if we attempt to patch that. 
So I think we need to make sure either both values are set, or both are not set. 

I'm guessing that's something we can do?

Reply:
It may be better to leave this PR until Dual Stack VIPs feature is implemented in the UI

Reply:
Ok!